src/js/themes/base.d.ts

```diff
@@ -1942,6 +1942,7 @@ export interface ThemeType {
       | { color?: ColorType }
       | { size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string };
     icon?: React.ReactNode | Icon;
+    responsive?: true;
```

Comment:
Should the type be a boolean?
```suggestion
    responsive?: boolean;
```

Reply:
good catch ☝️